cpp/ql/test/query-tests/Security/CWE/CWE-497/SAMATE/tests.c

```diff
@@ -67,6 +67,6 @@ void CWE535_Info_Exposure_Shell_Error__w32_char_01_bad()
             printLine("Unable to login.");
         }
         /* FLAW: Write sensitive data to stderr */
-        fprintf(stderr, "User attempted access with password: %s\n", password);
+        fprintf(stderr, "User attempted access with password: %s\n", password); // [NOT DETECTED]
```

Comment:
This is unfortunate, but I'm a bit surprised we don't treat `fprintf` arguments as a sink. It might be worth a separate query for specifically the password-to-console case.

Reply:
Quick prototype: https://lgtm.com/query/7564791297006366678/

Results are thin but do exist.  I agree this should be a separate query.